app/controllers/api/v1/token_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class TokenController < ApiController
+      skip_before_action :require_valid_jwt!, only: :request_token
+
+      # POST /api/v1/token/request
+      def request_token
+        @user = User.find_by(email: params[:email])
+
+        if @user && @user.valid_password?(params[:password])
+          render json: Jwt::Generate.new(user_id: @user.id), status: :created
+        else
+          render json: { error: I18n.t('errors.messages.wrong_credentials') }, status: :unprocessable_entity
+        end
+      end
+
+      # POST /api/v1/token/refresh
+      def refresh_token
+        render json: Jwt::Renew.new(bearer_token).generate!.to_json, status: :created
```

Comment:
`.to_json` isn't necessary here. You can use the same strategy that you did on `#request_token`